setup/symfony_server.rst

```diff
@@ -0,0 +1,271 @@
+Symfony Local Web Server
+========================
+
+You can run Symfony applications with any web server (Apache, nginx, the
+internal PHP web server, etc.) However, Symfony provides its own web server to
+make you more productive while developing your apps.
+
+Although this server is not intended for production use, it supports HTTP/2, TLS/SSL,
+automatic generation of security certificates, local domains, and many other
+features that sooner or later you'll need when developing web projects.
+
```

Comment:
Would it make sense to mention that this web server is not tied to Symfony, but that one can use it on any PHP app and also on plain HTML/SPA applications?